### ISSUE TITLE
Request: configuration for Python projects

### ISSUE THREAD

**Issue by jamesbraza** [DavidAnson/markdownlint-cli2#584]
I maintain a few Python projects and want to add `markdownlint-cli2` via `pre-commit`. However, I don't want to add a one-off file to the repo root centering on configuring `markdownlint-cli2`. I tried something like this with `pre-commit` (see the `args`):

```yaml
repos:
  - repo: https://github.com/DavidAnson/markdownlint-cli2
    rev: v0.18.1
    hooks:
      - id: markdownlint-cli2
        args: ["--config", '{"MD013":false}']
```

But that doesn't work, I get this message:

```none
Error: Unable to use configuration file '/path/to/ldp/{"MD013":false}'; Configuration file should be one of the supported names (e.g., '.markdownlint-cli2.jsonc') or a prefix with a supported name (e.g., 'example.markdownlint-cli2.jsonc').
```

With Python tooling, there is a common file to configure many tools: `pyproject.toml`. Here is an example, "find" for `tool.` to get an idea: https://github.com/Future-House/ldp/blob/v0.28.0/pyproject.toml

The request is to support better configuration for Python end use, either supporting configuration from:
- The CLI (without an accompanying config file)
- A `pyproject.toml`'s `tool.markdownlint` section

**Comment by jamesbraza** [comment]
Actually I just discovered https://github.com/DavidAnson/markdownlint-cli2/issues/7#issuecomment-695812069

> CLI2 by design does not support parameters on the command line. Every command line argument is treated as a glob pattern and all of the configuration is done through files.

I realized I should use CLI (1). Then I discovered https://github.com/igorshubovych/markdownlint-cli/issues/113 and realized this is implemented there. Case closed ✅
